src/docs/product/dashboards/widget-library/index.mdx

```diff
@@ -69,26 +69,23 @@ The chart now shows cumulative counts at different response time thresholds.
 
 ![Response Threshold Widget.](dashboard-response-threshold-widget.png)
 
-### Top issues
+### My top issues
 
-You can track the impact of your issues by counting the error events associated with each unique issue. Modify the "Issues For Review" widget by selecting the "All Events" data set. Update the search condition to filter for error events:
+You can track the impact of issues assigned to you or your team by counting the error events associated with each unique issue. Modify the "Issues For Review" widget by updating the search condition to filter for issue assignment:
 
-- Data Set: `All Events`
-- Search condition: `event.type:error`
+- Search condition: `is:unresolved is:for_review assigned_or_suggested:me`
 
-Update columns to group by `issues` and apply `count()` as the aggregating function. Order count in descending order so the widget displays issues with the most error events:
+Update columns to add `project` so you can see the associated project with each issue, and modify the “sort by” to `Events` so the widget displays issues with the most error events in descending order:
 
-- Columns: `issue, count(), any(message)`
-- Sort by: `count() desc`
+- Columns: `issue, assignee, events, title, project`
+- Sort by: `Events`
 
-### Unique issues by projects
+### Priority issues
 
-Monitoring the number of unique issues alongside the total number of errors in your projects can give you a sense of the impact of issues in your project. For example, a low number of unique issues and a high number of total errors indicates you have a few high impact issues. Modify the "Issues For Review" widget by selecting the "All Events" data set. Update the search condition to filter to error events:
+The issues data set is unique in that you can add a free form [token](https://docs.sentry.io/product/sentry-basics/search/#syntax) to your search condition that will filter by issue titles containing the token. For example, you can search `integrity` to filter by all issue titles containing integrity. Update the search condition with a token of your choice:
```

Comment:
Should we also code block the last reference to `integrity` as well? I feel like it's a special term so it needs some sort of highlighting. Maybe quotes would be better if not code?
```suggestion
The issues data set is unique in that you can add a free form [token](https://docs.sentry.io/product/sentry-basics/search/#syntax) to your search condition that will filter by issue titles containing the token. For example, you can search `integrity` to filter by all issue titles containing `integrity`. Update the search condition with a token of your choice:
```